packages/patternfly-3/patternfly-react/src/components/Toolbar/Toolbar.js

```diff
@@ -19,7 +19,16 @@ const ContextualToolbar = ({ children, className, ...props }) => {
       <Grid fluid className={className}>
         <Grid.Row className="toolbar-pf">
           <Grid.Col sm={12}>
-            <form className="toolbar-pf-actions">{toolbarChildren}</form>
+            <form
+              className="toolbar-pf-actions"
+              onSubmit={e => {
```

Comment:
maybe just passing `onSubmit` function as a prop is a tad bit more flexible? or spreading `...props` on the  `form`... both alternatives. It looks like we also pass `preventDefault` as a prop on the `Nav` in PF4 so not opposed to this method at all.

Reply:
I'm ok either way as well

Reply:
I'm leaning towards using `preventDefault` since there is an existing known pattern in `Nav` in PF4 -thanks